.github/DISCUSSION_TEMPLATE/feature-requests.yml

```diff
@@ -0,0 +1,8 @@
+body:
```

Comment:
One of the maintainers with `write` access must update the "discussion categories". [^docs]

The Discussion categories are the items you see in the sidebar when you're on the Discussions tab:

![renovate-discussion-categories](https://github.com/renovatebot/renovate/assets/34918129/07669024-1c80-4e1d-9b3b-3c4cb73eda51)

[^docs]: [GitHub Docs, Managing categories for discussions](https://docs.github.com/en/discussions/managing-discussions-for-your-community/managing-categories-for-discussions)

Reply:
![image](https://github.com/renovatebot/renovate/assets/6311784/21e21b63-8a69-4050-8bb3-6f0846b4411d)

WDYT?

Reply:
I see GitHub now has "sections" and "categories". We have no sections right now and I'm not sure if we'd benefit from them

Reply:
Now I'm confused because we still have this:

![image](https://github.com/renovatebot/renovate/assets/6311784/0ebe9209-0f89-4ac8-a83a-1e61372e78d0)

Reply:
## About the new categories

> ![image](https://user-images.githubusercontent.com/6311784/241722840-21e21b63-8a69-4050-8bb3-6f0846b4411d.png)
> 
> WDYT?

I like the angry emoji (😡) for the bug report category. 🤣 

## We don't need sections

> I see GitHub now has "sections" and "categories". We have no sections right now and I'm not sure if we'd benefit from them

We only have 3 categories, so we don't need sections.

## YML file confusion

> Now I'm confused because we still have this:
> 
> ![image](https://user-images.githubusercontent.com/6311784/241723275-0ebe9209-0f89-4ac8-a83a-1e61372e78d0.png)

Yeah, that's confusing me a bit as well! If I'm reading the docs right: the name of the `.yml` file must match the slug of the discussion category. Here's the full quote from the GitHub Docs: [^docs]

> ### Creating discussion category forms
>
> People with write access to a repository can create a discussion category form.
>
> 1. Navigate to the repository where you want to create a discussion category form. 
> 2. In your repository, create a file called `/.github/DISCUSSION_TEMPLATE/FORM-NAME.yml`, replacing `FORM-NAME` with the name for your discussion category form. The name must correspond with the slug for one of your discussion categories. For example, the template for the "Announcements" category should be `.github/DISCUSSION_TEMPLATE/announcements.yml`. For more information about creating new files on GitHub, see "[Creating new files](https://docs.github.com/en/repositories/working-with-files/managing-files/creating-new-files)."
> 3. In the body of the new file, type the contents of your discussion category form. For more information, see "[Syntax for discussion category forms](https://docs.github.com/en/discussions/managing-discussions-for-your-community/syntax-for-discussion-category-forms)."
> 4. Commit your file to the default branch of your repository. For more information, see "[Creating new files](https://docs.github.com/en/repositories/working-with-files/managing-files/creating-new-files)."

[^docs]: https://docs.github.com/en/discussions/managing-discussions-for-your-community/creating-discussion-category-forms#creating-discussion-category-forms